dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/OpenTelemetryProducer.java

```diff
@@ -95,6 +99,11 @@ public OpenTelemetry getOpenTelemetry() {
         //By default, MicroProfile Telemetry tracing is off.
         //The absence of an installed SDK is a “no-op” API
         //Operations on a Tracer, or on Spans have no side effects and do nothing
+        ComponentMetaData cData = ComponentMetaDataAccessorImpl.getComponentMetaDataAccessor().getComponentMetaData();
+        String applicationName = cData.getJ2EEName().getApplication();
+        String msg = Tr.formatMessage(tc, "CWMOT5100.tracing.is.disabled", applicationName);
+        Tr.info(tc, msg);
```

Comment:
Sorry, missed this earlier, you don't need to format the message first when you're using `Tr.info` since it will do the message lookup anyway.
```suggestion
        Tr.info(tc, "CWMOT5100.tracing.is.disabled", applicationName);
```

If we're getting the message for an exception message then we do need to call `Tr.formatMessage`